pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -3521,8 +3521,8 @@ let
         mktplcRef = {
           name = "uiua-vscode";
           publisher = "uiua-lang";
-          version = "0.0.19";
-          sha256 = "sha256-Tww1urq6CfLma254Sn5lwOYwbvxAeDZuBuFBQlzks1c=";
+          version = "0.0.21";
+          sha256 = "sha256-u57U/MmxvionFZp/tLK/KpddaxA/SUffeggKBSzmsXo=";
         };
         meta = {
           description = "VSCode language extension for Uiua";
```

Comment:
This probably was included by mistake

Reply:
This was on purpose, because it complements the main uiua update. (I could move it to another PR if you wanted to)

Reply:
Does uiua-vscode always get bumped together with uiua by kaikalii? Perhaps we should add a comment to both that they should be upgraded together then?

Reply:
I think it usually does, but IIRC with a little bit of delay.

Reply:
welp, it's probably fine for now